LayoutKitTests/TextViewLayoutTests.swift

```diff
@@ -0,0 +1,152 @@
+// Copyright 2016 LinkedIn Corp.
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+
+import XCTest
+@testable import LayoutKit
+
+class TextViewLayoutTests: XCTestCase {
+
+    func testNeedsView() {
+        let layout = TextViewLayout(text: "hi").arrangement().makeViews()
+        XCTAssertNotNil(layout as? UITextView)
+    }
+
+    func testTextViewLayout() {
+        for textTestCase in Text.testCases {
+            let textView = UITextView(text: textTestCase.text,
+                                      font: textTestCase.font)
```

Comment:
For the sake of consistency, please see https://github.com/linkedin/swift-style-guide 1.10 for the recommended indentation style.